official/resnet/keras/resnet56.py

```diff
@@ -0,0 +1,302 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please rename file to something like:  resnet_cifar_model.

That is what it is and it will not always be just be Resnet56 as resnet should be configurable to be a variety of layers very easily.